app/views/devise/passwords/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<h2>Change your password</h2>
```

Comment:
Actually on second thought I think I prefer your. Maybe make the button simply say `Submit`? Something though, the half-symmetry is killing me